presto-docs/src/main/sphinx/release/release-0.212.rst

```diff
@@ -0,0 +1,29 @@
+=============
+Release 0.212
+=============
+
+General Changes
+---------------
+
+* Add the :func:`ST_LineString` function to form a ``LineString`` from an array of points.
+* Enhanced :func:`ST_ConvexHull` and :func:`convex_hull_agg` functions to support geometry collections.
+* Improve performance for some query plans that include ``MarkDistinct`` or ``DistinctLimit``.
+* Remove ``round(x, d)`` and ``truncate(x, d)`` functions where ``d`` is a ``BIGINT`` (:issue:`11462`).
+* Change filtered global aggregations to include an explicit filter operator.
+* Fix :func:`ST_GeomFromBinary` function when applied to more than one row.
+* Fix unaccounted memory usage in broadcast join in source stage.
+* Fix occasional query failures when running ``EXPLAIN ANALYZE``.
+
+Hive Connector Changes
+----------------------
+
+* Prevent ORC writer from writing stripes larger than the max configured size for some rare data
+  patterns (:issue:`11526`).
+* Add sanity checks that fail queries if statistics read from the metastore are corrupt. Apply
+  sanity checks to statistics read from the Metastore. Corrupt statistics will instead be ignored
```

Comment:
The second sentence seems redundant. I'm guessing this was rephrased but the author forgot to delete it.

Reply:
```
Corrupt statistics can be ignored by setting the ``hive.ignore-corrupted-statistics``
configuration property or the ``ignore_corrupted_statistics`` session property.